### PR TITLE
adding web-data connector info

### PR DESCRIPTION
**Pull request by jbalis-usgs**
Adding info page for web data connector
-----------

Jira Ticket:
https://internal.cida.usgs.gov/jira/browse/SI-88

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 - Added an about page
 - Updated to the latest wdfn-viz package which uses USWDS 2.0.x
 - Added a separate docker-compose service to separate building the base from running the hugo command.
+- Added a content page for the NWISWeb Tableau Data Connector
```
Comment (jkreft-usgs) [inline CHANGELOG.md]: Put this in a new ### Added under ## Unreleased

File: content/connector.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+date: 2019-07-31
+slug: connector
```
Comment (jkreft-usgs) [inline content/connector.md]: change the slug to `about-connector`

File: content/connector.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+
+- What does this system provide ?
+This system provides USGS Water data and an accompanying Tableau schema for each parameter for each site, including geographic coordinates and datetimes for each observation. When invoking this system through the Tableau web data connector option, the data is automatically loaded into Tableau in the format specified by the generated schema.
+
+
+
+Questions
+==========
+Questions or comments about the Water for the Nation Projects? [here.](https://water.usgs.gov/contact/gsanswers?pemail=gs-w-ks_NWISWeb_Data_Inquiries&subject=Site+Number%3A+07144100&viewnote=%3CH1%3EUSGS+NWIS+Feedback+Request%3C%2FH1%3E%3Cp%3E%3Cb%3EPlease+enter+a+subject+in+the+form+below+that+briefly+summarizes+your+request%3C%2Fb%3E%3C%2Fp%3E)
```
Comment (jkreft-usgs) [inline content/connector.md]: Mention opening an issue on the github repo?

File: content/connector.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  - Tableau
+  - Web
+  - Data
+  - Connector
+---
+
+# NWISWeb Tableau Web Data Connector
+
+- How to use the NWISWeb Tableau Web Data Connector
+This system requires a desktop Tableau client. To use this system, first install a Tableau desktop client on your PC then enter the URL for this page under the web data connector option in the connect menu. Details of the constraints imposed by the database that feeds this tool are available here.
```
Comment (jkreft-usgs) [inline content/connector.md]: Sorry, one more thing.  You probably want to put the url of the actual connector in (I think we settled on labs.waterdata.usgs.gov/tableau-connector)

Comment (jbalis-usgs) [reply content/connector.md]: sounds good, I'll add that.

**Comment by jbalis-usgs** [comment]
In case there is ambiguity, I don't have the account permissions on this repository to confirm this pull request manually.

**Comment by jkreft-usgs** [comment]
@ayan-usgs you can merge this at your leisure

**Pull request merged**